projects/GKCore/GDModel/GDMSourceCitation.cs

```diff
@@ -48,20 +48,11 @@ public GDMSourceCitationData Data
         public GDMLines Description

```

Comment:
Only relevant usage of this method was already cordoned by `!IsPointer` (in `GEDCOMChecker.TransformSourceCitation`). Other are in tests and it is not a pointer there.